scripts/msct_parser.py

```diff
@@ -246,9 +246,9 @@ def checkFolderCreation(self, param):
         else:
             result_creation = 0  # no need for checking
         if result_creation == 2:
-            sct.printv("ERROR: Permission denied for folder creation...", type="error")
+            raise OSError("Permission denied for folder creation {}".format(param))
         elif result_creation == 1:
-            sct.printv("Folder " + param + " has been created.", 0, type='warning')
+            sct.log.warning("Folder " + param + " has been created.", 0)
```

Comment:
Creating a folder does not need to be a warning. printv without the warning should be fine. But due to the legacy code throwing a warning, I'm not sure what the motivation is.

Reply:
lets do `log.info`...